_opcodes/op108.html

```diff
@@ -11,8 +11,10 @@
 pst: 1
 ---
 Sets the individual reputation of the targeted creature(s) to the value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field.
-<br/>This does not affect the main reputation of the Party. Though it can affect ranger/paladinhood of any target.
+<br/>This does not affect the main reputation of the Party - This is not true for the EEs. Though it can affect ranger/paladinhood of any target.
```

Comment:
overcapitalized; would be more clearly stated as ", unless Type 3 or 4 are used".